CHANGELOG.md

```diff
@@ -27,6 +27,8 @@ The types of changes are:
 * Parallelize CI safe checks to reduce run time [#717](https://github.com/ethyca/fidesops/pull/717)
 * Add dependabot to keep dependencies up to date [#718](https://github.com/ethyca/fidesops/pull/718)
 
```

Comment:
Have left a blank space here in anticipation of [a conflict with this PR](https://github.com/ethyca/fidesops/pull/767/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR29)